common/init.sls

```diff
@@ -6,10 +6,24 @@ python2:
       - python
 
 {% if grains['os'] == 'Ubuntu' %}
-python2-dev:
+ubuntu-packages:
```

Comment:
Also, the new pkg.installed state should `require` the debconf state for ordering purposes.